PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -54,14 +55,78 @@
         android:layout_alignParentRight="true"
         android:layout_below="@id/scenarioNameEditText"
         android:src="@drawable/marco" />
-
+<RelativeLayout
+    android:id="@+id/power_bar"
+    android:layout_width="match_parent"
+    android:layout_height="75dp"
+    android:layout_alignParentBottom="true">
     <ImageView
         android:id="@+id/powerBarView"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="@dimen/power_bar_view_height"
         android:layout_alignParentBottom="true"
         android:layout_alignParentLeft="true"
-        android:background="@drawable/clothing_room_all" />
+        android:background="@drawable/power_status_bar"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true" />
+    <com.akexorcist.roundcornerprogressbar.RoundCornerProgressBar
+        android:id="@+id/power_bar1"
+        android:layout_width="@dimen/power_bar_width"
+        android:layout_height="@dimen/power_bar_height"
+        app:rcBackgroundPadding="@dimen/power_bar_radius_padding"
+        android:layout_marginRight="@dimen/power_bar_margin_right"
+        app:rcRadius="@dimen/power_bar_radius"
+        android:layout_alignParentRight="true"
+        android:layout_marginTop="@dimen/power_bar_margin_top"
+        app:rcBackgroundColor="@color/black"
+        app:rcProgressColor="@color/purple_light"
+        app:rcProgress="75"
+        app:rcMax="100"
```

Comment:
I haven't put the progress anywhere in dimens file as they Re dummy values and will anyway be set to start from 0 through code, this was just for visual effect. Let me know of I need to put them in dimens.